src/shared/src/language/texts/nb.ts

```diff
@@ -110,6 +110,8 @@ export function nb() {
       close: 'Lukk',
       contains: 'Inneholder',
       control_submit: 'Kontroller og send inn',
+      column_header_edit: 'Header ikon for redigerings-knapp kolonne',
```

Comment:
Maybe place the texts in the `group` section and not general?  

@Febakke any input on the texts themself?

Reply:
Agreed, i'll place them in `group` instead 👍